docs/index.bs

```diff
@@ -1903,7 +1903,7 @@ spec: webappsec-referrer-policy; urlPrefix: https://w3c.github.io/webappsec-refe
                     Note: The cache commit is allowed when the response's body is fully received.
 
             1. Add |responsePromise| to |responsePromises|.
-        1. Let |p| be [=waiting for all=] of |responsePromises|.
+        1. Let |p| be [=get a promise for waiting for all=] of |responsePromises|.
```

Comment:
The hyperlink doesn't seem added in the preview. Does it have to be [=waiting-for-all-promise=] to link to https://www.w3.org/2001/tag/doc/promises-guide#waiting-for-all-promise.

Also, based on the sample text at https://www.w3.org/2001/tag/doc/promises-guide#example-batch-request, should we say "Let |p| be the result of getting a promise to wait for all". Should we use that instead of "Let |p| be get a promise for waiting for all"?

Reply:
https://heycam.github.io/webidl/#ref-for-wait-for-all - the correct text is "get a promise to wait for all". Filed https://github.com/heycam/webidl/issues/787 as this is pretty confusing/misleading.